assets/js/modules/idea-hub/index.js

```diff
@@ -58,6 +58,10 @@ export const registerModule = ifIdeaHubIsEnabled( ( modules ) => {
 } );
 
 export const registerWidgets = ifIdeaHubIsEnabled( ( widgets ) => {
+	if ( ! widgets ) {
+		return;
+	}
```

Comment:
Is this because we're loading the Idea Hub module on this page but not Widgets?

Reply:
Yes, correct.

Reply:
See my comment regarding `googlesitekit-modules-idea-hub` – we shouldn't need to load it so this would also be unnecessary I think in that case.